bitwarden/values.yaml

```diff
@@ -0,0 +1,17 @@
+deploy_version: 0.1.6
+image_tag: latest
+
+applications:
+  - name: bitwarden
+    image: bitwardenrs/server-mysql
+    tag: "1.12.0"
+    secret: bitwarden
+    ingress:
+      hosts:
+        - host: bitwarden.pennlabs.org
+          paths: ["/"]
+    extraEnv:
+      - name: SIGNUPS_ALLOWED
+        value: false
+      - name: ENABLE_DB_WAL
+        value: false
```

Comment:
Because of how my templating works, I don't think this is an issue, but put quotes around false just to be safe.

Reply:
confirmed: this will work fine. good practice to have the quotes though since this syntax will not work if copy/pasted directly into a k8s `env:` block

Reply:
By definitely not already pushing to production. I can tell you that's not an issue.